spec/takeaway_spec.rb

```diff
@@ -0,0 +1,54 @@
+require 'takeaway'
+
+describe Takeaway do
+
+  let(:subject)   { Takeaway.new( '13', order, txt) }
+  let(:order)     { double(:order) }
+  let(:txt)       { double(:order) }
+  let(:num)       { '13' }
+  let(:name)      { 'oooh' }
+  let(:order_1)   { ['dohnuts', 4] }
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.